test/test_utils/__init__.py

```diff
@@ -1774,13 +1779,13 @@ def get_tensorflow_model_base_path(image_uri):
     return model_base_path
 
 
-def build_tensorflow_inference_command_tf27_and_above(model_name):
+def build_tensorflow_inference_command_tf27_and_above(model_name, entrypoint="/usr/bin/tf_serving_entrypoint.sh"):
```

Comment:
This goes along the comment in test_telemetry. Youare adding even more logic to handle a file location/name situation where all that can be handled in the dockerfile on the "COPY" line, making what file you are copying, use the destination path/name which is already standard for the DLC Framework.

Reply:
Same as above.